addons/mail/static/src/core/common/message.js

```diff
@@ -401,6 +407,23 @@ export class Message extends Component {
     /** @param {HTMLElement} bodyEl */
     prepareMessageBody(bodyEl) {}
 
+    prettifyThreadMentions() {
+        if (this.messageBody.el) {
+            const linkEls = this.messageBody.el.getElementsByTagName("a");
+            for (const linkEl of linkEls) {
+                if (linkEl.classList.contains("o_channel_redirect")) {
+                    const linkText = linkEl.innerHTML.substring(1);
+                    const iconClass = linkEl.classList.contains("o_sub_channel")
```

Comment:
I would name other classname `o_channel_redirect_asThread`:
- `o_sub_channel` could easily be picked by custom odoo classname, `o_channel_redirect` is heavily reserved for channel mentions so we can cautious with it. It'd best to not have too many short classnames that we need to watch out. Using same prefix `o_channel_redirect` helps understand this is an extension/modifier on this classname so less chances for conflict with some other code
- name of part `asThread` helps better see it is a sort of modifier/variant, and the name is more explicit while also using the end-user wording "thread" which I want us to use in priority over "sub-channel" which is more convenience to not rename technical "Thread" in whole discuss code.